src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -171,6 +171,7 @@ public void onUserUpdateActivityOrder(@Nonnull UserUpdateActivityOrderEvent even
     public void onUserTyping(@Nonnull UserTypingEvent event) {}
     public void onUserActivityStart(@Nonnull UserActivityStartEvent event) {}
     public void onUserActivityEnd(@Nonnull UserActivityEndEvent event) {}
+    public void onUserUpdateFlagsEvent(@Nonnull UserUpdateFlagsEvent event) {}
```

Comment:
```suggestion
    public void onUserUpdateFlags(@Nonnull UserUpdateFlagsEvent event) {}
```

Follow the naming convention